### ISSUE TITLE
check why mlflow is failing

### ISSUE THREAD

**Issue by edublancas** [ploomber/projects#76]
our mlflow example is failing: https://github.com/ploomber/projects/actions/runs/3790775714/jobs/6445671521

**Comment by RodolfoFerro** [comment]
I replicated the error by running mlflow in a local environment. The error drops when running the following command: 
```sh
$ mlflow experiments list
```

Returning the following:
```
Usage: mlflow experiments [OPTIONS] COMMAND [ARGS]...
Try 'mlflow experiments --help' for help.

Error: No such command 'list'.
```

I found that the error is caused by the following:
- I checked the docs and the previous versions before 2.0.1 (e.g. 1.13) use the command `list`(as seen [here in the docs](https://mlflow.org/docs/1.30.0/cli.html#mlflow-experiments-list))
- Newer versions (2.0.1 and above) use the command `search` (as specified [here](https://mlflow.org/docs/latest/cli.html#mlflow-experiments-search))

What'd be better? @edublancas I can work from here once I get a suggestion about this.
- Option 1: Search and replace commands to the new version
- Option 2: Specify the version to a previous one
